package.json

```diff
@@ -73,7 +77,7 @@
     "eslint-config-prettier": "^9.0.0",
     "eslint-config-standard": "^17.0.0",
     "eslint-plugin-import": "^2.22.1",
-    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-n": "^16.6.2",
```

Comment:
How come this has been introduced in this PR?

Some concerns around stability from https://github.com/finos/git-proxy/pull/486 that I want to make sure we have taken care of before we merge.

Reply:
removed

Reply:
reverted